src/AuthorizationCodeHandler.ts

```diff
@@ -25,24 +26,33 @@ class AuthorizationCodeHandler implements LambdaInterface {
 
 	@metrics.logMetrics({ throwOnEmptyMetrics: false, captureColdStartMetric: true })
 	async handler(event: APIGatewayProxyEvent, context: any): Promise<Response> {
+
+		// clear PersistentLogAttributes set by any previous invocation, and add lambda context for this invocation
+		logger.setPersistentLogAttributes({});
+		logger.addContext(context);
+
 		switch (event.resource) {
 			case ResourcesEnum.AUTHORIZATION:
 				if (event.httpMethod === HttpVerbsEnum.GET) {
 					let sessionId;
 					try {
-						logger.info("Event received", { event });
+						logger.info("Received authorization request", { requestId: event.requestContext.requestId });
```

Comment:
the `requestId` is redundant here as it is included by `addContext(context)`

Reply:
`function_requestId` seems to be available via `addContext` but not the one inside the api gateway event, hence keeping it 
<img width="1062" alt="Screenshot 2023-07-03 at 11 26 24" src="https://github.com/alphagov/di-ipv-cri-f2f-api/assets/26764987/0adebcdb-5a0b-442d-9d3a-096c22c586a9">